Content.Server/GameTicking/Rules/AntagLoadProfileRuleSystem.cs

```diff
@@ -31,14 +31,16 @@ private void OnSelectEntity(Entity<AntagLoadProfileRuleComponent> ent, ref Antag
             ? _prefs.GetPreferences(args.Session.UserId).SelectedCharacter as HumanoidCharacterProfile
             : HumanoidCharacterProfile.RandomWithSpecies();
 
-        SpeciesPrototype? species;
-        if (ent.Comp.SpeciesOverride != null)
+
+        if (profile?.Species is not { } speciesId || !_proto.TryIndex(speciesId, out var species))
         {
-            species = _proto.Index(ent.Comp.SpeciesOverride.Value);
+            species = _proto.Index<SpeciesPrototype>(SharedHumanoidAppearanceSystem.DefaultSpecies);
         }
-        else if (profile?.Species is not { } speciesId || !_proto.TryIndex(speciesId, out species))
+
+        if (ent.Comp.SpeciesOverride != null
+            && (ent.Comp.SpeciesOverrideBlacklist?.Contains(new ProtoId<SpeciesPrototype>(species.ID)) ?? false))
```

Comment:
When I implemented this as a whitelist I followed PJBs suggestion so you might want to check with her on turning it into a blacklist

Reply:
The use-case of temporarily disabling a species is better served by a blacklist rather than a whitelist due to the small number of disabled species compared to enabled ones.

Reply:
We won't overrule PJB's word, but honestly a blacklist makes sense here given that it's predominantly edge cases that can end up being problematic

Reply:
yea

Reply:
If a blacklist is problematic then we can just swap to using a whitelist in a later PR, the functionality is the same they only difference is how it's presented in YAML.